packages/components/src/components/hds/code-editor/index.ts

```diff
@@ -28,8 +32,14 @@ export interface HdsCodeEditorSignature {
   Blocks: {
     default: [
       {
-        Title?: ComponentLike<HdsCodeEditorTitleSignature>;
-        Description?: ComponentLike<HdsCodeEditorDescriptionSignature>;
+        Title?: WithBoundArgs<
+          typeof HdsCodeEditorTitle,
+          'onInsert' | 'editorId'
```

Comment:
[nit] same order as in the template code
```suggestion
          'editorId' | 'onInsert'
```